Authentication/Validators/TextInputValidatorType.swift

```diff
@@ -15,6 +15,10 @@ public enum ValidationResult {
         return .Invalid(errors: [errorMessage])
     }
     
+    public static func valid() -> ValidationResult {
```

Comment:
If we want to do what the new Swift guidelines are doing this should probably be a readonly property.